spack/packages/dla-future/package.py

```diff
@@ -26,7 +26,10 @@ class DlaFuture(CMakePackage, CudaPackage):
     depends_on("blaspp")
     depends_on("lapackpp")
     depends_on("hpx cxxstd=14 networking=none")
-    depends_on("hpx@1.5.0", when="~cuda")
+    # it would have been "more correct" specifying @1.5 as constraint but, if clingo
+    # solver works as expected, the default solver has problems with that.
+    # Eventually, we opted for specifying the range more explicitly, so it works with both
+    depends_on("hpx@1.5.0:1.5.1", when="~cuda")
```

Comment:
```suggestion
    depends_on("hpx@1.5.0:1.5.999", when="~cuda")
```
Just my 2 cents: it's horrible, but assuming hpx respects semver any future patch version should be ok, and whenever the open ranges PR lands in spack, it'll be easier to grep for `depends_on.*99` and fix it